### PR TITLE
Play Class

### PR DESCRIPTION
**Pull request by johnsonjo4531**
This commit will resolve issue #8. Tests are included. It passes StyleCop.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/sh4ne/Scrabble_CS3450/pull/39" target="_blank">ReviewNinja

File: Scrabble/Play.cs [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        /// returns playerID member.
+        /// </summary>
+        /// <returns>The playerID for the player making the move.</returns>
+        public int GetPlayerID()
+        {
+            return this.playerID;
+        }
+
+        /// <summary>
+        /// returns playerID member.
```
Comment (sh4ne) [inline Scrabble/Play.cs]: This is a copy and paste error. The provided summary is not for `GetXCoordinate`, but for `GetPlayerId`.

File: Scrabble/Play.cs [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+        /// </summary>
+        /// <param name="index">The index at this value of xCoordinates member will be returned.</param>
+        /// <returns>The index at the sent in value of xCoordinates member will be returned</returns>
+        public int GetXCoordinate(int index)
+        {
+            return this.coordinatesX[index];
+        }
+
+        /// <summary>
+        /// returns playerID member.
```
Comment (sh4ne) [inline Scrabble/Play.cs]: This is a copy and paste error as well here.

File: Scrabble/Play.cs [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+        /// </summary>
+        /// <param name="index">The index at this value of yCoordinates will be returned.</param>
+        /// <returns>The index at the sent in value of yCoordinates member will be returned</returns>
+        public int GetYCoordinate(int index)
+        {
+            return this.coordinatesY[index];
+        }
+
+        /// <summary>
+        /// returns playerID member.
```
Comment (sh4ne) [inline Scrabble/Play.cs]: And here as well.

File: Scrabble/Play.cs [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+        /// </summary>
+        /// <param name="index">The index at this value of letterTiles member will be returned.</param>
+        /// <returns>The index at the sent in value of letterTiles member will be returned</returns>
+        public LetterTile GetLetterTile(int index)
+        {
+            return this.letterTiles[index];
+        }
+
+        /// <summary>
+        /// returns playerID member.
```
Comment (sh4ne) [inline Scrabble/Play.cs]: Also here.

**Comment by sh4ne** [comment]
:+1: The comments I made aren't critical. So, I'll accept the request. If you fix the now though, you won't forget to do them.

**Comment by Peter-Scheelke** [comment]
:+1: Looks good to me.

**Comment by joshcv2112** [comment]
:+1:  Looks great.

**Comment by sh4ne** [comment]
You have conflicts now. Also, everyone needs to review their own files. Don't know if everyone knew that. Thanks.

**Comment by johnsonjo4531** [comment]
:+1: I added a new commit to fix copy paste errors and merge conflicts.

**Comment by johnsonjo4531** [comment]
I think you guys will have to +1 it again. It looks like it needs 3 stars still.

**Comment by sh4ne** [comment]
merged it in.

**Pull request merged**